### ISSUE TITLE
receiver swamped

### ISSUE THREAD

**Issue by alto777** [scottgchin/delta5_race_timer#91]
I am experimenting with the Delta5 concept and I observe that the RSSI from my receiver achieves a maximum output with the transmitter some distnace away - there is nothing like a peak to detect, it is a plateau.

So I came looking for a way to desensitize the receiver. Since I know nothing about RF, my experiments were only amusing failures. Never mind.

Or find sympathy, or assurances that it is just the truth. The implication would seem to be that Delta5 is only satifactory on larger tracks?

TIA

a7

**Comment by scottgchin** [comment]
It works on smaller tracks, but it needs to be tuned down so its not so sensitive.  For your issue though it sounds like something else.  Do the receivers have the SPI mod done?  on my receivers, with nothing on they avg between 70-90 for RSSI.  When a VTX is right over the receivers, it jumps to around 180 or so.

**Comment by alto777** [comment]
Yes I have the SPI. In fact, the modules I purchased were already SPI'd - opened one to discover that, then just wired the other expecting and finding that SPI is fine.

I do see variance in the RSSI depending on VTX distance from the receiver.

I see 90 background and 220 when I am very close. I see 220 from some distance, only at larger distances does it fall. Obviously it can only fall off to the background radiation. It nevers goes above a certain number, leading me to conclude the receiver is swamped.

How does one "tune down"? I was looking to reduce receiver sensitivity, perhaps you are suggesting looking at a lower or higher slightly frequency? Any suggestions for a starting offset if that was what you mean? I can experiment. Right now of course I am tuning exactly and only the numbers that are in the freq table and correspond to channels.

Step by step. THX.

a7
